test/test_latex_renderer.py

```diff
@@ -51,12 +51,20 @@ def test_image(self):
         self._test_token('Image', output, src='src')
 
     def test_link(self):
-        output = '\\href{target}{inner}'
-        self._test_token('Link', output, target='target')
+        for target, output in {
```

Comment:
TODO for me: `output` should be `expected` - hopefully the last overlooked renaming within #185. I will do this for the whole file in a separate commit.

Reply:
Done.